pkg/service/flipt/transport/service.go

```diff
@@ -23,10 +25,10 @@ import (
 
 const (
 	requestID   = "requestID"
-	defaultAddr = "localhost:9000"
+	defaultAddr = "http://localhost:8080"
```

Comment:
I know we're breaking a lot of the API, however, I wonder if we should keep this default as it was.
Just because changing this wouldn't cause a compile error. It would just cause a runtime error if FLipt wasn't available there, but was available by gRPC and that was your expectation.

Reply:
@GeorgeMac Well this would be the default for the new `transport` package. So it will only be `http` unless you strictly specify `grpc` so that was the contract I was going for, if that makes sense?

Reply:
I see what you're saying. The package rename means the contract is already broken.
Sounds good 👍